hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -204,6 +226,11 @@ instance IsTx CardanoTx where
   txId = getTxId . getTxBody
   balance = foldMap (\(TxOut _ value _) -> txOutValueToValue value)
 
+getOutputs :: CardanoTx -> [TxOut CtxTx Era]
+getOutputs tx =
+  let TxBody TxBodyContent{txOuts} = getTxBody tx
+   in txOuts
+
```

Comment:
What's your heuristic of ordering functions within this module? I see this interleaved between two `instance` terms?

Reply:
I was just annoyed by fourmulu's formatting of the header documentation `-- *`; and I first went to add some doc string to that function like `-- | Get outputs of a transaction` which feels utterly redundant with the name and type signature as well, so I figured I'd just move it in a place where it doesn't get a sticky `-- *` :grimacing: